docs/api/graphql_queries.md

```diff
@@ -216,18 +240,54 @@ Response:
 
 ## Querying Locations
 
-!!! tip
+You can get the Location object from any item by querying for `_location` or `_allLocations`.
+When you use `_location`, the API returns:
 
-    To make use of enhanced Location handling, you can add the beta 3.0 version of [ezplatform-graphql](https://github.com/ezsystems/ezplatform-graphql/tree/3.0) to your project.
+- the Location specified in the `locationId` or `urlAlias` argument
+- the Location based on the current SiteAccess
+- the main Location.
 
-    See [overview of the upcoming changes](https://github.com/ezsystems/ezplatform-graphql/pull/90).
+```
+{
+  content {
+    folder (contentId: 133) {
+      _allLocations {
+        pathString
+      }
+    }
+  }
+}
+```
 
-To query a Location and its children, use the repository schema:
+Response:
 
 ```
 {
-  _repository {
-    location(locationId: 2) {
+  "data": {
+    "content": {
+      "folder": {
+        "_allLocations": [
+          {
+            "pathString": "/1/2/128/132/"
+          },
+          {
+            "pathString": "/1/2/133/"
+          }
+        ]
+      }
+    }
+  }
+}
+```
+
+## Getting children of a Location
+
+To get a [Location's](#querying-locations) children, query the `children` property of an `item` or `content` object:
```

Comment:
I am not a big fan of documenting that approach as a first-class citizen for getting children. The API it exposes differs from the `item` one (since it is based on "location" and then "content", not "item"), and it will make queries weird.

In addition, the "location" depends on quite a few factors (siteaccess, to begin with, but also the choice of the location if there are several in the current site's tree root).

For these reasons, I'd strongly suggest to use query fields rather than the "raw" children API, as these will use proper queries and can take into account the dynamic location handling introduced by that change.